openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -63,6 +66,11 @@ def create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=None):
     client = oauth_adapter.get_client_for_token(access_token)
 
     jwt_expires_in = _get_jwt_access_token_expire_seconds()
+    try:
+        grant_type = access_token.application.authorization_grant_type
+    except Exception:
+        grant_type = ''
+        log.exception('Unable to get grant_type from access token.')
```

Comment:
Note: if we don't get this in the logs by the time we are going to merge the final PR, we could remove this try/except.